lib/node/NodeOutputFileSystem.js

```diff
@@ -2,16 +2,21 @@
 	MIT License http://www.opensource.org/licenses/mit-license.php
 	Author Tobias Koppers @sokra
 */
-var fs = require("fs");
-var path = require("path");
-var mkdirp = require("mkdirp");
+"use strict";
 
-function NodeOutputFileSystem() {}
-module.exports = NodeOutputFileSystem;
+const fs = require("fs");
+const path = require("path");
+const mkdirp = require("mkdirp");
+
+class NodeOutputFileSystem {
+	constructor() {
+		this.mkdirp = mkdirp;
+		this.mkdir = fs.mkdir.bind(this);
```

Comment:
cc/ @sokra Can you answer this question, please? https://github.com/webpack/webpack/pull/3901#discussion_r96791351